python/tvm/autotvm/measure/measure.py

```diff
@@ -55,37 +45,24 @@ class MeasureErrorNo(object):
     FLEET_ERROR = 6           # error of measure infrastructure
 
 
-def measure_option(mode,
+def measure_option(measure_func,
                    number=1,
                    repeat=1,
                    timeout=60,
                    parallel_num=1,
-                   pack_size=1,
+                   do_fork=True,
```

Comment:
again, do_fork can be part of measre_func(the previously no-fork argument) since it only applies to local

Reply:
do_fork is an argument of local_executor. It can also be applied to RPC mode.